client/src/components/overview/ImgGallery/Large.jsx

```diff
@@ -81,8 +91,8 @@ function Large(props) {
       <Minis minis={allImgs} currentImg={currentImg} onClickThu={onClickThu} leftClicked={leftClicked} rightClicked={rightClicked}/>
       <AllDefaultView className="alldefaultview">
         <DefaultView className="defaultview" src={currentImg.url} alt={defaultStyle.name}>
-          <LeftArrow onClick={leftButtonOnClick} type="button" data-testid="leftArrowImgGallery">&#8592;</LeftArrow>
-          <RightArrow type="button" data-testid="rightArrowImgGallery" onClick={rightButtonOnClick}>&#8594;</RightArrow>
+          <LeftArrow onClick={leftButtonOnClick} type="button" data-testid="leftArrowImgGallery" leftArrow={leftArrow}>&#xab;</LeftArrow>
+          <RightArrow type="button" data-testid="rightArrowImgGallery" onClick={rightButtonOnClick} rightArrow={rightArrow}>&#xbb;</RightArrow>
```

Comment:
Should we consolidate LeftArrow and RightArrow into a single component? They share a lot of the same attributes/props        

Something along the lines of: 
```
<Arrow data-testid="leftArrowImgGallery" onClick={leftButtonOnClick} active={leftArrow}>&#xab;</Arrow>
<Arrow data-testid="rightArrowImgGallery" onClick={rightButtonOnClick} active={rightArrow}>&#xbb;</Arrow>
``` 


nit: I think it would help readability if we made the attributes of LeftArrow and RightArrow follow the same order

i.e. `onClick`, should be in the same position for both

Reply:
Thanks for the input John! There's one styling difference between left and right arrow components, so I'm not sure how I would handle that if I combined them. Let me know if you have ideas of how to deal with that

Reply:
... I have no idea I was just trying to start smart LOL